server/svix-server/tests/e2e_attempt.rs

```diff
@@ -145,3 +150,130 @@ async fn test_list_attempts_by_endpoint() {
     receiver_1.jh.abort();
     receiver_2.jh.abort();
 }
+
+async fn test_attempt_retries(endpoint: &str, expected_status: i16) {
```

Comment:
Oh, I thought this was a test (obviously it isn't) because of the `test_` prefix. Maybe change that?